Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -192,7 +217,7 @@ def get_indicators_command(client, args):
     limit = int(args.get('limit'))
     indicators_list = fetch_indicators_command(client, itype)
     entry_result = camelize(indicators_list[:limit])
-    hr = tableToMarkdown('Indicators', entry_result, headers=['Value', 'Type', 'Rawjson'])
+    hr = tableToMarkdown('Indicators', entry_result, headers=['value', 'type', 'rawJSON'])
```

Comment:
does that come out ok when printing the rawJSON?

Reply:
what do you mean?

Reply:
will it print the values properly if thats a large dictionary?

Reply:
hmm probably not. Should I remove it from the HR?

Reply:
Yeah I think so, in the worst case we will add it back later